staging/src/github.com/kubeedge/beehive/pkg/core/core.go

```diff
@@ -20,7 +21,8 @@ func StartModules() {
 		beehiveContext.AddModule(name)
 		// Assemble typeChannels for sendToGroup
 		beehiveContext.AddModuleGroup(name, module.Group())
-		go module.Start()
```

Comment:
Thanks for the fixing, and if the module panic, the whole process should exit, so it's not reasonable to recover it here.